modern-business/sass/site/header/_site-header.scss

```diff
@@ -24,6 +24,14 @@
 			margin-bottom: 2rem;
 		}
 	}
+
+	.fse-site-logo {
+		margin-bottom: 0;
+        img {
```

Comment:
Nit: this is happening because some of these indents are spaces vs. tabs.

Reply:
sorted :-) - well that little issue is, now I just need to fix VS Code's format on save for these files so I don't need to manually add indents and get it wrong.